master/internal/core.go

```diff
@@ -484,23 +484,37 @@ func (m *Master) Run() error {
 	})
 
 	// Elm WebUI.
-	m.echo.File("/", filepath.Join(webuiRoot, "elm/public/index.html"))
 	m.echo.File("/ui", filepath.Join(webuiRoot, "elm/public/index.html"))
```

Comment:
blocking: Expand the same structure to the Elm routes and `Static` calls.

Reply:
was thinking maybe also mix some of the initialization but separate looks a bit nicer.